### PR TITLE
IAM-382 Add dashboard to Login UI

### PR DESCRIPTION
**Pull request by bencekov**
This commit adds the dashboard to Login UI operator. The spec for the dashboard can be found in the Engineering drive.
The visualizations for the alerts concerning a lack of valid urls to working Hydra and Kratos deployments will be patched in after the deep health check is patched into the login UI.

The Login UI dashboard shows the alerts for Kratos and Hydra as well.

# TESTING

To see login UI dashboard, you will need to deploy the cos bundle:
```
juju deploy cos-lite --trust
```

Pack and deploy identity platform login ui operator (in the operator root directory):

```
charmcraft pack
juju deploy ./identity*.charm  --resource oci-image=$(yq eval '.resources.oci-image.upstream-source' metadata.yaml) --trust
```

# Integrations:

You need to make the following integrations:

```
juju integrate identity-platform-login-ui grafana
```

After this you may access grafana's UI with your browser. The url is grafana-ip:3000. Username is admin, you can get the password with `juju run grafana/0 get-admin-password`

On the grafana interface you should be able to find the dashboards in the menu on the left side of the screen. The dashboard's name is Identity Platform Overview Dashboard.

### CODE REVIEW

File: src/grafana_dashboards/identity-platform-login-ui-operator.json.tmpl [hunk 1]
```diff
@@ -0,0 +1,1181 @@
...
+          "format": "time_series",
+          "instant": true,
+          "interval": "",
+          "intervalFactor": 2,
+          "legendFormat": "{{juju_unit}}",
+          "refId": "A",
+          "step": 60
+        }
+      ],
+      "title": "Hydra Unit Availability",
```
Comment (natalian98) [inline src/grafana_dashboards/identity-platform-login-ui-operator.json.tmpl:265]: Do we care about Hydra in Login UI if it has its own dashboard?

**Review by natalian98 — COMMENTED** [review]
> You need to make the following integrations:
 `juju integrate identity-platform-login-ui grafana`

> After this you may access grafana's UI with your browser. The url is grafana-ip:3000. Username is admin, you can get the password with `juju run grafana/0 get-admin-password`

I suppose you meant grafana-k8s? grafana is a machine charm.

**Review by shipperizer — DISMISSED** [review]
:shipit:

even though the query might be slightly off (see pr #48) I think they can be patched subsequently

File: src/grafana_dashboards/identity-platform-login-ui-operator.json.tmpl (hunk 1 above)
Comment (shipperizer) [reply src/grafana_dashboards/identity-platform-login-ui-operator.json.tmpl:265]: Login UI will be the aggregator dashboard as right now it seems a bundle doesn't have any "property"

so idea is that login UI dashboard will be the IAM platform dashboard

**Comment by bencekov** [comment]
> > You need to make the following integrations:
> > `juju integrate identity-platform-login-ui grafana`
> 
> > After this you may access grafana's UI with your browser. The url is grafana-ip:3000. Username is admin, you can get the password with `juju run grafana/0 get-admin-password`
> 
> I suppose you meant grafana-k8s? grafana is a machine charm.

when you deploy the bundle, it will appear as grafana in your model.

**Review by shipperizer — APPROVED** [review]

**Pull request merged**